src/cljs/rems/fields.cljs

```diff
@@ -167,7 +167,7 @@
 (defn- event-value [event]
   (.. event -target -value))
 
-(defn text-field
+(defn phone-text-field
```

Comment:
this looks like an unused function

Reply:
Yes, I should have removed it while refactroing, my mistake

Reply:
I very often forget these things, I am too used to eslint throwing warnings about it that I cannot get from cloure tools unfortunately

Reply:
addressed